tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -117,6 +117,29 @@ def test_map_coordinates_int(self, xp, scp, dtype):
         return out
 
 
+@testing.parameterize(*testing.product({
+    'order': [0, 1, 2, 3, 4, 5],
+    'mode': ['constant', 'nearest', 'mirror'] + scipy16_modes,
```

Comment:
probably have to drop the `scipy16_modes` here and the `_condition_scipy_version_skip` below for the 8.x branch

(and the same for `TestZoomOutputSize1` too)

Reply:
The comment above was intended to apply to a backport PR (if one is planned)